virtcontainers/utils/utils_linux.go

```diff
@@ -67,6 +67,7 @@ func FindContextID() (*os.File, uint64, error) {
 		return nil, 0, err
 	}
 
+	defer vsockFd.Close()
```

Comment:
No, this is **wrong**! vsockfd needs to keep open so that we don't race vsock cid allocation with other qemu processes.

Reply:
What about the Close call on line 85 ?